### PR TITLE
 Update the 'Updating a cluster using the CLI' section

### PR DESCRIPTION
**Pull request by xenolinux**
OCP version: 4.6+
SME @LalatenduMohanty 
QE @jiajliu 
[Preview](https://deploy-preview-45571--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-cli.html)
No bug is assigned

### CODE REVIEW

**Comment by jiajliu** [comment]
lgtm

File: updating/updating-cluster-cli.adoc [hunk 1]
```diff
@@ -11,7 +11,7 @@ All OpenShift advisories link directly to this assembly. If you are doing work t
 But if you really need to, please contact the release notes team so they can change the advisory templates. These templates are not part of the openshift-docs repo.
 ////
 
-You can update, or upgrade, an {product-title} cluster within a minor version by using the OpenShift CLI (`oc`).
+You can update, or upgrade, an {product-title} cluster within a minor version by using the OpenShift CLI (`oc`). You can use the same instructions for updating a cluster between minor versions.
```
Comment (opayne1) [inline updating/updating-cluster-cli.adoc:14]: Hi @xenolinux this looks repetitive of the first sentence. Both state you can "update or upgrade a cluster within a minor version" the way I interpret it. Could you explain the need for adding the additional sentence? Thank you!

**Review by opayne1 — COMMENTED** [review]
Just small comment!

File: updating/updating-cluster-cli.adoc (hunk 1 above)
Comment (xenolinux) [reply updating/updating-cluster-cli.adoc:14]: Yes. The first sentence states that users can update/upgrade only *within a minor version*. This sentence will help users to understand that they can update/upgrade a cluster *within* a minor version as well as *between* minor versions using the CLI procedure. Earlier this wasn't clearer.

Comment (opayne1) [reply updating/updating-cluster-cli.adoc:14]: Or something like: 
```suggestion
You can update, or upgrade, an {product-title} cluster within a minor version by using the OpenShift CLI (`oc`). You can also update a cluster between minor versions by following the same instructions.
```
I am still not sure about "following the same instructions" though.

Comment (wking) [reply updating/updating-cluster-cli.adoc:14]: This wording is already in [the web-console page][1], so whatever we do, we should be consistent between the web-console and `oc` approaches.

Personally I'd go with something like:

> You can use the OpenShift CLI (`oc`) to update, or upgrade, an {product-title} cluster.  The following procedure is the same for updates between and within minor versions.

But all of the alternatives discussed earlier in this thread are fine with me too.

[1]: https://docs.openshift.com/container-platform/4.10/updating/updating-cluster-within-minor.html

**Review by tmalove — COMMENTED** [review]
/LGTM

**Pull request merged**